dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/MvcTestConfig.java

```diff
@@ -115,7 +125,11 @@ public CustomRequestMappingHandlerMapping requestMappingHandlerMapping()
         CustomRequestMappingHandlerMapping mapping = new CustomRequestMappingHandlerMapping();
         mapping.setOrder( 0 );
         mapping.setContentNegotiationManager( manager );
-
+        TestInterceptorRegistry registry = new TestInterceptorRegistry();
+        addInterceptors( registry );
+        registry.addInterceptor( new ConversionServiceExposingInterceptor( mvcConversionService ) );
+        registry.addInterceptor( new ResourceUrlProviderExposingInterceptor( mvcResourceUrlProvider ) );
+        mapping.setInterceptors( registry.getInterceptors().toArray() );
```

Comment:
FYI: This is a work-around due to limitations in spring as detailed in https://github.com/spring-projects/spring-boot/issues/5004 - We need a subclass of the registry to be able to access its `getInterceptors()` method and we need to make a copy of the code that is found the the spring `WebMvcConfigurationSupport` class initialization code. 

Since this only affects the test config this seems the lesser evil. So with this fix interceptors will now work in spring mvc tests as well.